src/java/test/org/apache/zookeeper/server/quorum/CommitProcessorConcurrencyTest.java

```diff
@@ -106,6 +106,7 @@ public void shutdown() {
 
                 @Override
                 public void notifyStopping(String threadName, int errorCode) {
+                    Assert.fail("Commit processor crashed " + errorCode);
```

Comment:
oh, I didn't see your test, sorry. I think there's a problem with the way you created the PR. did you run "git commit" twice ? github shows two commits here - see the commits tab. I'm also new to git, but the way I usually do it is first time the PR is created I run
> git add -u
> git commit -m 'JIRA number and description'
> git push

then if I want to change something I do:
> git add -u
> git amend
> git push -f

Now I think you should remove one of the commits from this PR. Please look online how to do that...